kubernetes/iac/modules/hw-tf-module-cluster-resources/apisix.tf

```diff
@@ -85,3 +90,48 @@ resource "helm_release" "apisix-dashboard" {
 
   namespace = "apisix"
 }
+
+resource "kubectl_manifest" "apisix_dapr_upstream" {
+  depends_on = [
+    helm_release.apisix
+  ]
+
+  yaml_body = <<YAML
+apiVersion: apisix.apache.org/v2
+kind: ApisixUpstream
+metadata:
+  name: ${local.apisix_name}-dapr
+  namespace: ${local.apisix_namespace}
+spec:
+  loadbalancer:
+    type: roundrobin
+  portLevelSettings:
+  - port: 3500
+    scheme: http
+  - port: 50001
+    scheme: grpc
+YAML
+}
+
+resource "kubectl_manifest" "apisix_dapr_route" {
+  depends_on = [
+    helm_release.apisix
+  ]
+
+  yaml_body = <<YAML
+apiVersion: apisix.apache.org/v2
+kind: ApisixRoute
+metadata:
+  name: ${local.apisix_name}-dapr
+  namespace: ${local.apisix_namespace}
+spec:
+  http:
```

Comment:
I don't really like this solution as it exposes the "dapr way" to the outside. It's better than having nothing running for now, but the API should be a gRPC (Web) API, not a dapr API.

Reply:
Me neither. The goal of this pull request is to be on the same level as with the previous HTTP approach. In the future we should not expose the "Dapr way" of gRPC, thats right. If mobile-app can adopt gRPC-web then we should not expose both, gRPC-web and gRPC.